package.json

```diff
@@ -12,56 +12,50 @@
       "name": "Rafal Chlodnicki (@rchl)"
     }
   ],
-  "files": [
-    "lib",
-    "types"
-  ],
   "publishConfig": {
     "access": "public"
   },
-  "main": "lib/module.js",
-  "types": "types/index.d.ts",
+  "type": "module",
+  "exports": {
+    ".": {
+      "import": "./dist/module.mjs",
+      "require": "./dist/module.cjs"
+    }
+  },
+  "main": "./dist/module.cjs",
+  "types": "./dist/types.d.ts",
+  "files": [
+    "dist",
+    "types"
+  ],
   "scripts": {
+    "prepare": "yarn dev:prepare",
+    "prepack": "nuxt-module-build",
```

Comment:
The `prepare` script causes problems when publishing as it runs after `prepack` but before the artifact is created so it reverts to stubbed build for development.